src/style.css

```diff
@@ -0,0 +1,19 @@
+body {
+  margin: 0;
+}
+
+div {
+  height: 100px;
+}
+
+#block_red {
+  background-color: red;
+}
+
+#block_green {
+  background-color: green;
+}
+
+#block_blue {
+  background-color: blue;
+}
```

Comment:
checklist https://github.com/mate-academy/layout_colored-blocks/blob/master/checklist.md

Reply:
It actually supposed to be classes, I forgot to change that =) now it's changed